great_expectations/core/expectation_suite.py

```diff
@@ -739,9 +739,19 @@ def add_expectation(
             overwrite_existing=overwrite_existing,
         )
 
+    @public_api
     def show_expectations_by_domain_type(self) -> None:
-        """
-        Displays "ExpectationConfiguration" list, grouped by "domain_type", in predetermined designated order.
+        """Displays "ExpectationConfiguration" list, grouped by "domain_type", in predetermined designated order.
+
+        The means of displaying is through the use of the "Pretty Print" library method "pprint.pprint()".
```

Comment:
Feels like an implementation detail? Good either way but don't think this is necessary?

Reply:
@cdkini I thought that it might be nice to highlight -- in case someone wants to PR against it with a nicer way...

Reply:
We don't want to capture the implementation detail though in our public API.